internal/cmd/run_test.go

```diff
@@ -29,11 +28,14 @@ import (
 
 	rootcmd "github.com/tetratelabs/getenvoy/internal/cmd"
 	"github.com/tetratelabs/getenvoy/internal/globals"
+	"github.com/tetratelabs/getenvoy/internal/moreos"
 	"github.com/tetratelabs/getenvoy/internal/test"
 	"github.com/tetratelabs/getenvoy/internal/test/morerequire"
 	"github.com/tetratelabs/getenvoy/internal/version"
 )
 
+const lf = moreos.LineSeparator
```

Comment:
what does "f" of lf mean here?

Reply:
renamed to ls!

Reply:
actually "ln" looks nicer as matches println